.env

```
@@ -0,0 +1,10 @@
+SMTP_HOST=smtp.ionos.mx
+SMTP_PORT=587
+SMTP_USER=c.melo@carlosmeloabogados.com
+SMTP_PASS=TU_SMTP_PASS
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SMTP_PASS key should go before the SMTP_PORT key

(UnorderedKey)